src/app.jsx

```diff
@@ -65,6 +65,7 @@ class Application extends React.Component {
             podmanRestartAvailable: false,
             currentUser: _("User"),
             privileged: false,
+            location: {},
```

Comment:
maybe it should be renamed to `url_parameters`?

Reply:
:man_shrugging: